tests/foreman/cli/test_oscap.py

```diff
@@ -611,11 +708,30 @@ def test_positive_associate_scap_policy_with_tailoringfiles_name(self):
 
         :expectedresults: The policy is created and associated successfully.
 
-        :caseautomation: notautomated
+        :caseautomation: automated
         """
+        file_name = settings.oscap.tailoring_path
+        file_name = file_name.split('/')[(file_name.split('/')).__len__() - 1]
+        ssh.upload_file(local_file=settings.oscap.tailoring_path,
+                        remote_file="/tmp/{0}".format(file_name))
+        tailoring_file = make_tailoringfile({
+            'scap-file': '/tmp/{0}'.format(file_name)
+        })
+        tailor_profile_id = tailoring_file['tailoring-file-profiles'][0]['id']
+        scap_policy = make_scap_policy({
+            'scap-content-id': self.scap_id_rhel7,
+            'scap-content-profile-id': self.scap_profile_id_rhel7,
+            'period': OSCAP_PERIOD['weekly'].lower(),
+            'weekday': OSCAP_WEEKDAY['friday'].lower(),
+            'tailoring-file': tailoring_file['name'],
+            'tailoring-file-profile-id': tailor_profile_id
+        })
+        self.assertEqual(scap_policy['tailoring-file-id'],
+                         tailoring_file['id'])
+        self.assertEqual(scap_policy['tailoring-file-profile-id'],
+                         tailor_profile_id)
 
     @run_only_on('sat')
-    @stubbed()
     @tier2
     def test_positive_create_scap_policy_without_hostgroups(self):
```

Comment:
it seems that case is covered in `test_postive_create_scap_policy_with_valid_name`, so please remove it entirely